exonum/benches/criterion/block.rs

```diff
@@ -298,7 +298,9 @@ mod cryptocurrency {
             if arg.seed % 2 == 0 {
                 Ok(())
             } else {
-                let error_kind = ErrorKind::Service { code: 15 };
+                let error_kind = ErrorKind::Service {
+                    local_error_code: 15,
```

Comment:
To avoid misunderstanding: in [ECR-3973] it is written that `ErrorKind::Runtime { code: 1 }` can be confused with `runtime_id`, so renaming is going to make errors more friendly to users.